### PR TITLE
Refactor front-end

### PR DESCRIPTION
**Pull request by Shico-UA**
Refactored graph.js file.

### CODE REVIEW

File: graph_creator/static/graph_creator/graphs.js [hunk 1]
```diff
@@ -1,92 +1,65 @@
...
+    appendChartData(courses_gd, courses, 'Active Courses');
+    appendChartData(students_gd, students, 'Active Students');
+
+    window.onresize = function() {
+        Plotly.Plots.resize(instances_gd);
+        Plotly.Plots.resize(courses_gd);
+        Plotly.Plots.resize(students_gd);
+    };
+
+})();
```
Comment (DmytryiStriletskyi) [inline graph_creator/static/graph_creator/graphs.js]: Newline.

File: static/graph_creator/graphs.js [hunk 2]
```diff
@@ -1,92 +1,65 @@
...
+    appendChartData(courses_gd, courses, 'Active Courses');
+    appendChartData(students_gd, students, 'Active Students');
+
+    window.onresize = function() {
+        Plotly.Plots.resize(instances_gd);
+        Plotly.Plots.resize(courses_gd);
+        Plotly.Plots.resize(students_gd);
+    };
+
+})();
```
Comment (DmytryiStriletskyi) [inline static/graph_creator/graphs.js]: Newline.

File: graph_creator/static/graph_creator/graphs.js (hunk 1 above)
Comment (Shico-UA) [inline graph_creator/static/graph_creator/graphs.js]: Done

File: static/graph_creator/graphs.js (hunk 2 above)
Comment (Shico-UA) [inline static/graph_creator/graphs.js]: Done

**Pull request merged**